CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,61 @@
+## Kodluyoruz Açık Kaynak | Davranış Kuralları
+
+Bu belge Kodluyoruz’un açık kaynak projelerine katkı sağlayanlar arasında kabul edilmiş kuralları tanımlamaktadır. 
+
+### Neden Bir Davranış Kurallarına İhtiyaç Duyuyoruz?
+
+Bu Davranış Kuralları (Code of Conduct), rahat, huzurlu, korkusuz ve üretken bir topluluk inşaa edebilmemize yardımcı olmak için tasarlanmıştır. Amacı; topluluğumuza bir takım gereksiz kurallar yüklemek, "kötü" insanlara cezalar uygulamak ya da geçmişte bir takım yanlış yapılmış uygulamaları düzeltmeye çalışmak değildir. Burada yazılmış kurallara uyulmayan ve/veya daha kötü şartlarda çalışmak durumunda kalmış insanlarla bir araya geldiğimizde de yine harika bir topluluk ortaya koyabilmeyi amaçlamaktadır.
+
+## Kurallar
+
+Projemize katkıda bulunan herkes aşağıdaki kurallara uymalıdır:
+
+### 1. Hoşgörü ve Saygı
+
+Projemize katkıda bulunan herkes, tüm insanları hoşgörüyle karşılamalı ve saygı göstermelidir. Herhangi bir kişiyi, cinsiyeti, yaşam tarzı, ten rengi, etnik kökeni, inancı veya herhangi bir diğer özelliği nedeniyle ayırt etmemelidir.
+
+### 2. Ayrımcılığı Reddetmek
+
+Projemize katkıda bulunan herkes, kişilerin rengi, maddi durumu, ırkı, dili-lehçesi, siyasi görüşü, inancı-inançsızlığı, cinsiyeti ve cinsel yönelimi nedeniyle birbirlerini ayrıştırmasını, gruplaşmasını ve bu doğrultuda tavır almasını kesin bir şekilde reddetmelidir.
+
+### 3. Ayrımcı ve Cinsiyetçi Söylemden Uzak Durmak
+
+Projemize katkıda bulunan herkes, cinsiyet temelli eşitsizliği ortaya koyabilecek mahiyette olan her türlü söyleme ve eyleme kesin bir dille karşı çıkmalıdır.
+
+### 4. Bilgi Eksikliği İçin Suçlamamak
+
+Projemize katkıda bulunan herkes, bir diğer kişiyi bilmediği bir konu yüzünden yargılamamalı ve kırmamaya özen göstermelidir. Herkesin farklı bilgi seviyeleri ve tecrübeleri olabilir. Bu nedenle, projemize katkıda bulunan herkes, birbirine yardımcı olmak ve birlikte öğrenmek için açık fikirli olmalıdır.
+
```

Comment:
### Bilgi Paylaşımı ve Öğretme Yaklaşımı

Projemize katkıda bulunan herkes, bilgi paylaşımı ve öğretme yaklaşımında, "bilmeyene öğretmek" ilkesine bağlı kalmalıdır. Projemize katılan öğrencilerin her biri farklı ilgi, tecrübe ve bilgi seviyelerine sahip olabilir. Bu nedenle, projemize katkıda bulunan herkes, başarılı öğrencilerin yanı sıra odaklanmakta zorluk çeken, motivasyonunu kaybetmiş ya da kaybetme aşamasına gelmiş gençlere de özel bir ilgi göstermelidir.

Kodluyoruz eğitmenleri, öğrencilere sadece belli kalıpları öğretmekle yetinmezler. Aksine, yazılımı sevdirmek ve öğretmek için öğrencilerin farklı ihtiyaçlarına özel bir ilgi gösterirler. Bu yaklaşım, projemize katkıda bulunan herkesin de benimsemesi gereken bir yaklaşımdır.

Bu nedenle, projemize katkıda bulunan herkes, bilgi paylaşımı ve öğretme sürecinde, farklı öğrenci ihtiyaçlarına özel bir ilgi göstermeli ve öğrencilerin motivasyonunu artırmak için çaba göstermelidir.

Reply:
Bu açıklamayı bölerek maddelere yaymışız. Eklememizin gerekli olmadığını düşünüyorum. 

Yaklaşımın nasıl yapılması gerektiği konusunun olmasının gerektiğini düşünüyorsak, içeride revize edilmesi gereken kısımlar var. Gerekli görülmesi durumunda önerileri yapabilirim. Eline sağlık 💯